### ISSUE TITLE
the DELETE endpoint will not delete inspection if there is no pictures

### ISSUE THREAD

**Issue by scaspers** [DanielCaspers/DigitalInspection#150]
It looks like it fails when it tries to find and delete the ...Uploads/Inspections/{ordernumber} directory that does not exist unless picture(s) have been uploaded.

`lert-danger">^  <i class="material-icons">error</i>^    <span>An unknown error o
ccurred at Inspections/Delete.^^Innermost exception: (System.IO.DirectoryNotFoun
dException) Could not find a part of the path &#39;/var/www/virtuals/daniel/di/U
ploads/Inspections/004787919&#39;.</span>^^(P)MT,(T)EL,(W)RK,(E)<button type="bu
tton" class="btn btn-default" onclick="window.location.reload()">^n" -H "Digita<
i class="material-icons md-light">refresh</i>^69036" -X POST htt</button>^</div>`

If I upload a picture then the DELETE will work and return 200

**Comment by scaspers** [comment]
Delete is now working however. after deleting a subsequent call to ReportForOrder will bring a 200 response with stack trace.  I think this behavior existed prior but question if it should return 404??

**Comment by scaspers** [comment]
Still an error if the workorder directory is not present in Uploads/Inspections (the directory can be empty and call will succeed)
New error stack attached.
[DELETE error.txt](https://github.com/DanielCaspers/DigitalInspection/files/15224812/DELETE.error.txt)

**Comment by DanielCaspers** [comment]
@scaspers is this still a problem?

**Comment by scaspers** [comment]
Yes, this is still broken.  Will only delete if there is a pictures saved.

` An unknown error occurred at Inspections/Delete.

Innermost exception: (System.IO.DirectoryNotFoundException) Could not find a part of the path &#39;/var/www/virtuals/murphyauto.net/domains/di.murphyauto.net/public_html/Uploads/Inspections/004807901&#39;.


        <pre><code>  at System.IO.FileSystem.RemoveDirectory (System.String fullPath, System.Boolean recursive) [0x00027] in &lt;5dfd69ae4e3b402db546d8ded6fc755e&gt;:0 
  at System.IO.Directory.Delete (System.String path, System.Boolean recursive) [0x00006] in &lt;5dfd69ae4e3b402db546d8ded6fc755e&gt;:0 
  at DigitalInspection.Services.Core.InspectionService.DeleteInspection (DigitalInspection.Models.ApplicationDbContext ctx, DigitalInspection.Models.Inspections.Inspection inspection) [0x001f6] in &lt;9526f337f4f64d38908a3896bffe59ab&gt;:0 
  at DigitalInspection.Controllers.InspectionsController.Delete (System.String workOrderId) [0x000c6] in &lt;9526f337f4f64d38908a3896bffe59ab&gt;:0 
  at (wrapper dynamic-method) System.Object.lambda_method(System.Runtime.CompilerServices.Closure,System.Web.Mvc.ControllerBase,object[])
  at System.Web.Mvc.ActionMethodDispatcher.Execute (System.Web.Mvc.ControllerBase controller, System.Object[] parameters) [0x00000] in &lt;cc73190bab9d435c831510ff295c572a&gt;:0 
  at System.Web.Mvc.ReflectedActionDescriptor.Execute (System.Web.Mvc.ControllerContext controllerContext, System.Collections.Generic.IDictionary`2[TKey,TValue] parameters) [0x00072] in &lt;cc73190bab9d435c831510ff295c572a&gt;:0 
  at System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod (System.Web.Mvc.ControllerContext controllerContext, System.Web.Mvc.ActionDescriptor actionDescriptor, System.Collections.Generic.IDictionary`2[TKey,TValue] parameters) [0x00000] in &lt;cc73190bab9d435c831510ff295c572a&gt;:0 
  at System.Web.Mvc.Async.AsyncControllerActionInvoker+ActionInvocation.InvokeSynchronousActionMethod () [0x00000] in &lt;cc73190bab9d435c831510ff295c572a&gt;:0 
  at System.Web.Mvc.Async.AsyncControllerActionInvoker.&lt;BeginInvokeSynchronousActionMethod&gt;b__39 (System.IAsyncResult asyncResult, System.Web.Mvc.Async.AsyncControllerActionInvoker+ActionInvocation innerInvokeState) [0x00000] in &lt;cc73190bab9d435c831510ff295c572a&gt;:0 
  at (wrapper delegate-invoke) System.Web.Mvc.Async.EndInvokeDelegate`2[System.Web.Mvc.Async.AsyncControllerActionInvoker+ActionInvocation,System.Web.Mvc.ActionResult].invoke_TResult_IAsyncResult_TState(System.IAsyncResult,System.Web.Mvc.Async.AsyncControllerActionInvoker/ActionInvocation)
  at System.Web.Mvc.Async.AsyncResultWrapper+WrappedAsyncResult`2[TResult,TState].CallEndDelegate (System.IAsyncResult asyncResult) [0x00000] in &lt;cc73190bab9d435c831510ff295c572a&gt;:0 
  at System.Web.Mvc.Async.AsyncResultWrapper+WrappedAsyncResultBase`1[TResult].End () [0x00029] in &lt;cc73190bab9d435c831510ff295c572a&gt;:0 
  at System.Web.Mvc.Async.AsyncResultWrapper.End[TResult] (System.IAsyncResult asyncResult, System.Object tag) [0x00007] in &lt;cc73190bab9d435c831510ff295c572a&gt;:0 
  at System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethod (System.IAsyncResult asyncResult) [0x00000] in &lt;cc73190bab9d435c831510ff295c572a&gt;:0 
  at System.Web.Mvc.Async.AsyncControllerActionInvoker+AsyncInvocationWithFilters.&lt;InvokeActionMethodFilterAsynchronouslyRecursive&gt;b__3d () [0x00014] in &lt;cc73190bab9d435c831510ff295c572a&gt;:0 
  at System.Web.Mvc.Async.AsyncControllerActionInvoker+AsyncInvocationWithFilters+&lt;&gt;c__DisplayClass46.&lt;InvokeActionMethodFilterAsynchronouslyRecursive&gt;b__3f () [0x00086] in &lt;cc73190bab9d435c831510ff295c572a&gt;:0 
  at System.Web.Mvc.Async.AsyncControllerActionInvoker+&lt;&gt;c__DisplayClass33.&lt;BeginInvokeActionMethodWithFilters&gt;b__32 (System.IAsyncResult asyncResult) [0x00000] in &lt;cc73190bab9d435c831510ff295c572a&gt;:0 
  at System.Web.Mvc.Async.AsyncResultWrapper+WrappedAsyncResult`1[TResult].CallEndDelegate (System.IAsyncResult asyncResult) [0x00000] in &lt;cc73190bab9d435c831510ff295c572a&gt;:0 
  at System.Web.Mvc.Async.AsyncResultWrapper+WrappedAsyncResultBase`1[TResult].End () [0x00029] in &lt;cc73190bab9d435c831510ff295c572a&gt;:0 
  at System.Web.Mvc.Async.AsyncResultWrapper.End[TResult] (System.IAsyncResult asyncResult, System.Object tag) [0x00007] in &lt;cc73190bab9d435c831510ff295c572a&gt;:0 
  at System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethodWithFilters (System.IAsyncResult asyncResult) [0x00000] in &lt;cc73190bab9d435c831510ff295c572a&gt;:0 
  at System.Web.Mvc.Async.AsyncControllerActionInvoker+&lt;&gt;c__DisplayClass21+&lt;&gt;c__DisplayClass2b.&lt;BeginInvokeAction&gt;b__1c () [0x00000] in &lt;cc73190bab9d435c831510ff295c572a&gt;:0 
  at System.Web.Mvc.Async.AsyncControllerActionInvoker+&lt;&gt;c__DisplayClass21.&lt;BeginInvokeAction&gt;b__1e (System.IAsyncResult asyncResult) [0x00000] in &lt;cc73190bab9d435c831510ff295c572a&gt;:0 </code></pre>
`
